google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -90,97 +106,184 @@ def sample_table_to_csv():
+    text: str
 
+
+@dataclasses.dataclass
+class Line:
+    """Represents a wrapped documentai.Document.Page.Line.
+
+    Attributes:
+        documentai_object (google.cloud.documentai.Document.Page.Line):
+            Required. The original google.cloud.documentai.Document.Page.Line object.
+        text (str):
+            Required. UTF-8 encoded text.
     """
 
-    header_rows = _table_rows_from_documentai_table_rows(
-        table_rows=list(documentai_table.header_rows), text=text
-    )
-    body_rows = _table_rows_from_documentai_table_rows(
-        table_rows=list(documentai_table.body_rows), text=text
+    documentai_object: documentai.Document.Page.Line = dataclasses.field(
+        init=True, repr=False
     )
+    text: str = dataclasses.field(init=True, repr=False)
+    _page: Page = dataclasses.field(init=True, repr=False)
 
-    return Table(
-        documentai_table=documentai_table, body_rows=body_rows, header_rows=header_rows
+    tokens: List[Token] = dataclasses.field(
+        init=False, repr=False, default_factory=list
     )
 
+    def __post_init__(self):
+        self.tokens = _get_children_of_element(
+            self.documentai_object, self._page.tokens
+        )
+
 
 @dataclasses.dataclass
-class Block:
-    """Represents a wrapped documentai.Document.Page.Block.
+class FormField:
+    """Represents a wrapped documentai.Document.Page.FormField.
 
     Attributes:
-        documentai_block (google.cloud.documentai.Document.Page.Block):
-            Required. The original google.cloud.documentai.Document.Page.Block object.
-        text (str):
-            Required. UTF-8 encoded text.
+        documentai_formfield (google.cloud.documentai.Document.Page.FormField):
+            Required. The original google.cloud.documentai.Document.Page.FormField object.
+        field_name (str):
+            Required. The form field name
+        field_value (str):
+            Required. The form field value
     """
 
-    documentai_block: documentai.Document.Page.Block
-    text: str
+    documentai_formfield: documentai.Document.Page.FormField
+    field_name: str
+    field_value: str
 
 
 @dataclasses.dataclass
 class Paragraph:
     """Represents a wrapped documentai.Document.Page.Paragraph.
 
     Attributes:
-        documentai_paragraph (google.cloud.documentai.Document.Page.Paragraph):
+        documentai_object (google.cloud.documentai.Document.Page.Paragraph):
             Required. The original google.cloud.documentai.Document.Page.Paragraph object.
         text (str):
             Required. UTF-8 encoded text.
     """
 
-    documentai_paragraph: documentai.Document.Page.Paragraph
-    text: str
+    documentai_object: documentai.Document.Page.Paragraph = dataclasses.field(
+        init=True, repr=False
+    )
+    text: str = dataclasses.field(init=True, repr=False)
+    _page: Page = dataclasses.field(init=True, repr=False)
+
+    lines: List[Line] = dataclasses.field(init=False, repr=False)
```

Comment:
Just a reminder: let's be sure to have one unit test for every element that verifies these children element fields are correctly populated.

Reply:
it's a little difficult to test the `Lines,Tokens etc` by themselves due to the `_page` addition but i did expand `test_Page` to check children

Reply:
Sounds good.